### ISSUE TITLE
"stacked create service xxx" not working anymore after updating from 1.2.0 to 1.3.0

### ISSUE THREAD

**Issue by felixkriechbaum** [Stacked-Org/stacked#866]
Hello together,

today i updated from Stacked-cli from 1.2.0 to 1.3.0 and now the create service comman is not working anymore

```
F:\Development\NextChatApp\chatclient> stacked create service user
Unhandled exception:
Invalid argument(s): Could not find an option named "template".
#0      ArgResults.[] (package:args/src/arg_results.dart:65:7)
#1      CreateServiceCommand.run (package:stacked_cli/src/commands/create/create_service_command.dart:46:37)
#2      CommandRunner.runCommand (package:args/command_runner.dart:212:27)
#3      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:122:25)
#4      new Future.sync (dart:async/future.dart:302:31)
#5      CommandRunner.run (package:args/command_runner.dart:122:14)
#6      main (file:///C:/Users/Felix/AppData/Local/Pub/Cache/hosted/pub.dev/stacked_cli-1.3.0/bin/stacked.dart:59:12)
<asynchronous suspension>
Unhandled exception:
Invalid argument(s): Could not find an option named "template".
#0      ArgResults.[] (package:args/src/arg_results.dart:65:7)
#1      CreateServiceCommand.run (package:stacked_cli/src/commands/create/create_service_command.dart:46:37)
#2      CommandRunner.runCommand (package:args/command_runner.dart:212:27)
#3      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:122:25)
#4      new Future.sync (dart:async/future.dart:302:31)
#5      CommandRunner.run (package:args/command_runner.dart:122:14)
#6      main (file:///C:/Users/Felix/AppData/Local/Pub/Cache/hosted/pub.dev/stacked_cli-1.3.0/bin/stacked.dart:59:12)
<asynchronous suspension>
```

I have already tried
* deactivate and reactivate the cli
* restart ide
* complete fresh project

After downgrading to 1.2.0 it works again

For people have the same problem
`dart pub global deactivate stacked_cli`
`dart pub global activate stacked_cli 1.2.0`

**Comment by FilledStacks** [comment]
Hey, I will start looking into this now.

**Comment by FilledStacks** [comment]
Fixed in 1.3.1
